ush/ioda/bufr2ioda/run_bufr2ioda.py

```diff
@@ -78,15 +103,18 @@ def bufr2ioda(current_cycle, RUN, DMPDIR, config_template_dir, COM_OBS):
 
         # use the bufr2ioda executable for the ob type
         bufr2iodaexe = BIN_GDAS + '/bufr2ioda.x'
-        cmd = Executable(bufr2iodaexe)
-        cmd.add_default_arg(yaml_output_file)
-        logger.info(f"Executing {cmd}")
-        cmd()
+
+        # append the values to the lists
+        yaml_files.append(yaml_output_file)
 
         # Check if the converter was successful
         # if os.path.exists(yaml_output_file):
         #     rm_p(yaml_output_file)
 
+    # run all bufr2ioda yamls in parallel
+    with mp.Pool(num_cores) as pool:
```

Comment:
Note, the easiest way to do this was to split them into python and YAML+executable groups. If they are roughly equal in size, that probably is okay, but we may want to combine them all into the same pool?